report_builder/models.py

```diff
@@ -138,6 +141,153 @@ def get_good_display_fields(self):
                 bad_display_fields.append(display_field)
         return display_fields.exclude(id__in=[o.id for o in bad_display_fields])
 
+    def report_to_list(self, queryset=None, user=None, preview=False):
```

Comment:
Same as comment above. If it just depends on the queryset and arguments, and doesn't mutate self, then make that a manager method https://docs.djangoproject.com/en/1.8/topics/db/managers/#modifying-initial-manager-querysets

Either way, the docstring should give a little better idea of what it does. What sort of list does it returns?